frontend/src/ui/pages/AboutPage/AboutPage.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react'
+import './AboutPage.scss'
+import Page1 from './Page1'
+import Page2 from './Page2'
+import Page3 from './Page3'
+import { Link } from 'react-router-dom'
+import arrow from '../../assets/other/arrow.png'
+
+function AboutPage() {
+	const [currentPage, setCurrentPage] = useState(1)
+
+	const handlePageChange = (pageNumber) => {
+		setCurrentPage(pageNumber)
+	}
+
+	const renderChatBubbles = () => {
+		switch (currentPage) {
+		case 1:
+			return (
+				<div>
```

Comment:
Do we need so many nested empty divs?
![image](https://github.com/kbojarowska/DoTheBeast/assets/75124863/e913af18-b9db-4676-b821-02c70222ba3f)